src/render/gpu/SDL_render_gpu.c

```diff
@@ -1689,6 +1689,9 @@ static bool GPU_CreateRenderer(SDL_Renderer *renderer, SDL_Window *window, SDL_P
         SDL_SetBooleanProperty(create_props, SDL_PROP_GPU_DEVICE_CREATE_DEBUGMODE_BOOLEAN, debug);
         SDL_SetBooleanProperty(create_props, SDL_PROP_GPU_DEVICE_CREATE_PREFERLOWPOWER_BOOLEAN, lowpower);
 
+        // We have our own software renderer, skip software-based Vulkan drivers
```

Comment:
Actually I don't think we want to do this. If somebody requests the GPU driver because they have shaders they want to use, for example, then we want to give it to them, even if it ends up being a slower software Vulkan implementation.

Reply:
Removed this commit, so now it's just the one unique commit!